.travis.yml

```diff
@@ -1,7 +1,25 @@
+sudo: required
+
 language: node_js
 node_js:
-  - "stable"
+- "node" 
+
+services:
+- docker
+
+after_success:
```

Comment:
Don't you think it'll make more sense to put `after_success` near the end of the file?

Reply:
Makes sense, moved